css/modules/content.scss

```diff
@@ -20,6 +20,12 @@
     .text_with_summary a,
     .text_long         a { color:$color-heading; text-decoration:underline; }
 
+    #news-section {
```

Comment:
Just a minor quibble:
This CSS styling looks like it really belongs in the news.scss, instead of this general content file.  That way, all the styling properties for those news boxes are in the same place.

Reply:
good eye, nuked all together!